met-web/src/components/engagement/admin/create/authoring/AuthoringBottomNav.tsx

```diff
@@ -141,7 +117,14 @@ const AuthoringBottomNav = (props: AuthoringBottomNavProps) => {
                                 marginLeft: 'auto',
                             }}
                         >
-                            <img style={{ paddingRight: '0.3rem' }} src={pagePreview} alt="page preview icon" />
+                            <img
```

Comment:
I'm not sure how to make it into an icon since this is a custom graphic that Jessica created for the button. I exported it from Figma and included it in our assets. If we have a process for converting images to icons then let me know and I will do it.

Reply:
I looked it up and apparently I can add alt="" and aria-hidden="true" in order to keep it invisible to a reader.

Reply:
I'm alright with this as a quick fix this time. Could you please just test it once you implement?